doc/default.nix

```diff
@@ -38,15 +38,87 @@ let
     docBook.id = "home-manager-options";
   };
 
+  nixosModuleDocs = nmd.buildModulesDocs {
+    modules = let
+      nixosModule = module: pkgs.path + "/nixos/modules" + module;
+      mockedNixos = with lib; {
+        options = {
+          environment.pathsToLink = mkOption {
+            type = types.listOf types.str;
+            visible = false;
+          };
+          systemd.services = mkOption {
+            type = types.attrsOf types.anything;
+            visible = false;
+          };
+          users.users = mkOption {
+            type = types.attrsOf types.anything;
+            visible = false;
+          };
+        };
+      };
+    in [
+      ../nixos/default.nix
+      mockedNixos
+      (nixosModule "/misc/assertions.nix")
+      scrubbedPkgsModule
+    ];
+    moduleRootPaths = [ ./.. ];
+    mkModuleUrl = path:
+      "https://github.com/nix-community/home-manager/blob/master/${path}#blob-path";
```

Comment:
Why not use a variable for the path prefix since it gets reused?

Reply:
Didn't quite finish up the PR. Was meaning to move the common stuff to a function. Have done so now 🙂